File: POOVeiculos/Python/Carro.py
from Veiculo import Veiculo

class Carro(Veiculo):
    def __init__(self, modelo, tipo, potencia, ano):
        super().__init__(modelo, tipo, potencia, ano)

    def __del__(self):
        print("Objeto Carro Destruído com Sucesso!")

File: POOVeiculos/C++/Carro.cpp
#include "Carro.h"

Carro::Carro(string modelo, string tipo, float potencia, int ano) : Veiculo(modelo, tipo, potencia, ano) {

}

Carro::~Carro() {
    cout << "Objeto Carro Destruído com Sucesso!" << endl;
}
